static/js/top_left_corner.js

```diff
@@ -76,9 +77,23 @@ export function narrow_to_recent_topics() {
     remove($(".top_left_private_messages"));
     remove($(".top_left_starred_messages"));
     remove($(".top_left_mentions"));
+    remove($(".top_left_inbox"));
     $(".top_left_recent_topics").addClass("active-filter");
     pm_list.close();
     setTimeout(() => {
         resize.resize_stream_filters_container();
     }, 0);
 }
+
+export function narrow_to_inbox() {
+    remove($(".top_left_all_messages"));
+    remove($(".top_left_private_messages"));
+    remove($(".top_left_starred_messages"));
+    remove($(".top_left_recent_topics"));
+    remove($(".top_left_mentions"));
+    $(".top_left_inbox").addClass("active-filter");
+    pm_list.close();
+    setTimeout(() => {
+        resize.resize_stream_filters_container();
+    }, 0);
+}
```

Comment:
It could be a good prep commit/PR to replace this coding pattern where we basically have a bunch of copies of `deselect_top_left_corner_items` each with one item removed -- is there a reason we can't just have these all call `deselect_top_left_corner_items` and then select the thing they want ... and also why `deselect_top_left_corner_items` couldn't just use a single selector to catch all the top-left corner things we want to deselect if currently selected?